spinta/datasets/commands/check.py

```diff
@@ -1,14 +1,45 @@
 from spinta import commands
 from spinta.components import Context
 from spinta.datasets.components import Dataset, Resource
+import re
```

Comment:
Standard library imports always goes to the top, see [PEP-8](https://pep8.org/#imports).